core/new-gui/src/app/workspace/service/dynamic-schema/schema-propagation/schema-propagation.service.ts

```diff
@@ -50,16 +54,21 @@ export class SchemaPropagationService {
       .merge(
         this.workflowActionService.getTexeraGraph().getLinkAddStream(),
         this.workflowActionService.getTexeraGraph().getLinkDeleteStream(),
-        this.workflowActionService.getTexeraGraph().getOperatorPropertyChangeStream()
-          .debounceTime(SCHEMA_PROPAGATION_DEBOUNCE_TIME_MS),
-        this.workflowActionService.getTexeraGraph().getDisabledOperatorsChangedStream(),
-        )
-      .flatMap(() => this.invokeSchemaPropagationAPI())
-      .filter(response => response.code === 0)
-      .subscribe(response => {
-        this.operatorInputSchemaMap = response.result;
-        this._applySchemaPropagationResult(this.operatorInputSchemaMap);
-      });
+        this.workflowActionService.getTexeraGraph().getOperatorPropertyChangeStream(),
+        this.workflowActionService.getTexeraGraph().getDisabledOperatorsChangedStream()
+      ).debounceTime(SCHEMA_PROPAGATION_DEBOUNCE_TIME_MS)
+      .mergeMap(() => this.invokeSchemaPropagationAPI().pipe(catchError(err => {
```

Comment:
as we discussed, add a comment explain why add an error handler in subscribe doesn't work